hs_core/hydroshare/utils.py

```diff
@@ -36,6 +37,7 @@
 logger = logging.getLogger(__name__)
 
 
+@lru_cache(maxsize=1)
```

Comment:
Using lr_cache with maxsize of 1 confused me for a moment. It now makes sense since the functions have no parameters and always return the same result.